client/ice-servers.go

```diff
@@ -19,7 +20,55 @@ import (
 )
 
 func formGenericIceServers(config *config.ICEConfig) (adapters.IceServersConfig, error) {
-	return adapters.IceServersConfig{}, nil
+	iceServers := []webrtc.ICEServer{}
+	if config.StunEnabled {
+		for proto, ports := range config.StunPorts {
+			for _, port := range ports {
+				stunProto := "stun"
+				if proto == "tls" {
+					stunProto = "stuns"
+				}
+				url := fmt.Sprintf("%s:%s:%d?transport=%s",
```

Comment:
Should we handle servers (e.g., [pion/turn](https://github.com/pion/stun/blob/master/uri.go#L21)) that are not supporting queries?

Reply:
How do you mean?

Reply:
My first idea was to create a URL without the query and one with the query.  

For example, we add two webrtc.ICEservers: one has `127.0.0.1:3478` while the other contains `127.0.0.1:3478?transport=udp`.  

I am not 100% sure about this solution since we give up fine control over the transport protocol (e.g., when they use the same port TCP and UDP).

Reply:
Sorry I got completely distracted due to travel on this one.... 

Ah interesting idea. I don't like the idea of adding two. I think we'd just add an extra flag in the config to say "don't add the transport to the uri params" - what do you think? I've not come across any turn servers that don't support queries or at least they just ignore them.

Reply:
Adding an extra config parameter makes sense to me. Perhaps we can name it `use_stun_rfc_uri` since [RFC7094](https://datatracker.ietf.org/doc/html/rfc7064#section-3.1) defines the URI without queries. Wdyt?

Reply:
Added in [26f6154](https://github.com/nimbleape/iceperf-agent/pull/13/commits/26f61540338cea3e8bbc93c9f9dfc772a3166492)